core/src/main/kotlin/io/rover/campaigns/core/events/domain/Event.kt

```diff
@@ -14,4 +14,15 @@ data class Event(
         name: String,
         attributes: Attributes
     ): this(name, attributes, Date(), UUID.randomUUID())
+
+    companion object {
+        fun screenViewed(screenName: String, contentName: String? = null, contentID: String? = null): Event {
```

Comment:
minor niggle, let's re-order it so contentID comes before contentName. I did the same in the iOS one